operate/cli.py

```diff
@@ -75,7 +82,7 @@ def __init__(
     ) -> None:
         """Initialize object."""
         super().__init__()
-        self._path = (home or (Path.cwd() / OPERATE)).resolve()
+        self._path = (home or OPERATE_HOME).resolve()
```

Comment:
Would the removal of Path.cwd not cause any issue?

Reply:
`OPERATE_HOME` is already defined as `Path.cwd() / OPERATE` in `operate/constants.py`